quisp/modules/QRSA/RuleEngine/RuleEngine.h

```diff
@@ -67,12 +67,15 @@ class RuleEngine : public IRuleEngine {
   IRealTimeController *realtime_controller;
   int *qnic_burst_trial_counter;
   qnicResources *allResources;  // Size will be defined in initialization. If 3 qnic types, then size is 3. Type defined in QUBIT.h
+  std::map<int, photonTransmissionConfig> transmission_interface;
+
   /*
    * DEFINED in QNIC.h
    * typedef std::multimap<int, StationaryQubit*> EntangledPairs;//entangled Node address -> pointer to that local qubit
    * typedef EntangledPairs* qnicResources;//For each qnic. If the number of "qnic" is 3, then the size is 3.
    * For resource management over.
    * */
+  bool DEBUG_flag = false;
```

Comment:
why do you need this?

Reply:
Ah, actually, we don't need this. Let me remove this.